src/darwin/Framework/CHIP/zap-generated/MTRBaseClusters.h

```diff
@@ -240,40 +282,95 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  * Cluster Scenes
- *    Attributes and commands for scene configuration and manipulation.
+ *
+ * Attributes and commands for scene configuration and manipulation.
  */
 @interface MTRBaseClusterScenes : MTRCluster
 
 - (instancetype _Nullable)initWithDevice:(MTRBaseDevice *)device
                               endpointID:(NSNumber *)endpointID
                                    queue:(dispatch_queue_t)queue NS_DESIGNATED_INITIALIZER MTR_NEWLY_AVAILABLE;
 
+/**
+ * Command AddScene
+ *
+ * Add a scene to the scene table. Extension field sets are supported, and are inputed as '{"ClusterID": VALUE,
```

Comment:
where do extension fields go into the command below? Basically wondering if this comment applies or is something that ZAP input xml describes but may confuse API readers.

Reply:
As we fix the scenes support, I expect we will fix this comment too... But yes, it's possible it will confuse people in the meantime.